src/webgl/p5.RendererGL.js

```diff
@@ -951,16 +960,14 @@ p5.RendererGL.prototype._getEmptyTexture = function() {
 };
 
 p5.RendererGL.prototype.getTexture = function(img) {
-  var checkSource = function(element) {
-    return element.src === img;
-  };
-  //this.drawMode = constants.TEXTURE;
-  var tex = this.textures.find(checkSource);
-  if (!tex) {
-    tex = new p5.Texture(this, img);
-    this.textures.push(tex);
+  var textures = this.textures;
+  for (var it = 0; it < textures.length; ++it) {
+    var texture = textures[it];
+    if (texture.src === img) return texture;
```

Comment:
using a for loop instead of `Array.find()` which is really slow..

Reply:
Maybe indexOf would be faster? http://www.andygup.net/fastest-way-to-find-an-item-in-a-javascript-array/

Reply:
i don't think indexOf would work in this case since we're looking for the element that has a particular (`src`) property. this is that `find()` is intended for - apply a predicate to each element and return the first that matches. but the function call is really slow.